docs/code-examples/mesh_simple.rs

```diff
@@ -0,0 +1,32 @@
+//! Log a simple colored triangle.
+use rerun::components::{Mesh3D, MeshId, RawMesh3D};
+use rerun::{MsgSender, RecordingStreamBuilder};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let (rec_stream, storage) = RecordingStreamBuilder::new("mesh").memory()?;
+
+    let mesh = RawMesh3D {
+        mesh_id: MeshId::random(),
+        vertex_positions: [[0.0, 0.0, 0.0], [1.0, 0.0, 0.0], [0.0, 1.0, 0.0]]
+            .into_iter()
+            .flatten()
+            .collect(),
+        indices: Some([0, 1, 2].into_iter().collect()),
+        vertex_normals: Some(
+            [[0.0, 0.0, 1.0], [0.0, 0.0, 1.0], [0.0, 0.0, 1.0]]
+                .into_iter()
+                .flatten()
+                .collect(),
+        ),
+        vertex_colors: Some([0xff000000, 0x00ff0000, 0x0000ff00].into_iter().collect()),
```

Comment:
```suggestion
        vertex_colors: Some([0xff0000ff, 0x00ff00ff, 0x0000ffff].into_iter().collect()),
```

Lucky we don't support alpha lol